cognite/client/data_classes/iam.py

```diff
@@ -336,23 +336,27 @@ def dump(self, camel_case: bool = True) -> dict[str, Any]:
         }
 
 
+SessionStatus: TypeAlias = Literal["ready", "active", "cancelled", "expired", "revoked", "access_lost"]
+SessionType: TypeAlias = Literal["CLIENT_CREDENTIALS", "TOKEN_EXCHANGE"]
```

Comment:
Looks like there might be a new one now, `ONESHOT_TOKEN_EXCHANGE`:
```suggestion
SessionType: TypeAlias = Literal["CLIENT_CREDENTIALS", "TOKEN_EXCHANGE", "ONESHOT_TOKEN_EXCHANGE"]
```

Reply:
Nice, forgot to check the spec again.